recipes/clean-up.rb

```diff
@@ -0,0 +1,7 @@
+
+handler_directory = ::File.join(Chef::Config[:file_cache_path], 'handler')
+
+directory handler_directory do
+  recursive true
+  action :delete
+end
```

Comment:
@jeremymv2 thanks for the honest feedback. That is very helpful! Background: This was an approach @vjeffrey and I discussed before. From my perspective it feels strange to install code on a node just for runtime and not cleaning the tmp stuff up. Nevertheless if that is not standard practice we should not do this approach.

@vjeffrey I propose we leave the handler files there for now, I've an idea how we can get around generating the file in future to get the best of both worlds. But lets try that in future PRs.